packages/block-library/src/post-comments/index.php

```diff
@@ -25,10 +25,23 @@ function render_block_core_post_comments( $attributes, $content, $block ) {
 	$post = get_post( $block->context['postId'] );
 	setup_postdata( $post );
 
-	// This generates a deprecate message.
-	// Ideally this deprecation is removed.
 	ob_start();
+	// There's a deprecation warning generated by WP Core.
+	// Ideally this deprecation is removed from Core.
+	// In the meantime, this removes it from the output.
+	add_filter(
+		'deprecated_file_trigger_error',
+		function() {
+			return false;
+		}
+	);
```

Comment:
Could we not just use `__return_false` here?

```suggestion
	add_filter( 'deprecated_file_trigger_error', '__return_false' );
```